test/fancylogger.py

```diff
@@ -412,7 +418,7 @@ def somefunction(self):
         handler = fancylogger.logToScreen()
         logger = fancylogger.getLogger(fname=False, clsname=False)
         logger.warn("blabla")
-        print stringfile.getvalue()
+        print(stringfile.getvalue())
```

Comment:
should import from future ...

Reply:
`import` is there now (but this change is still needed, `print 'foo'` never works in Python 3, even with the `import`